airbyte-integrations/connectors/source-mailgun/source_mailgun/manifest.yaml

```diff
@@ -0,0 +1,78 @@
+version: "0.29.0"
+
+definitions:
+  selector:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_path: ["items"]
+
+  requester:
+    type: HttpRequester
+    url_base: |
+      {{
+        'https://api.mailgun.net/v3'
+        if config['domain_region'] == 'US'
+        else 'https://api.eu.mailgun.net/v3'
+      }}
+    http_method: "GET"
+    authenticator:
+      type: "BasicHttpAuthenticator"
+      username: "api"
+      password: "{{ config['private_key'] }}"
+
+  retriever:
+    type: SimpleRetriever
+    record_selector:
+      $ref: "#/definitions/selector"
+    paginator:
+      type: NoPagination
```

Comment:
Please add pagination as it was implemented before https://github.com/airbytehq/airbyte/blob/b78ad4e9d905f2f02631f7897acabf6441375032/airbyte-integrations/connectors/source-mailgun/source_mailgun/source.py#L42-L49